community/tutorials/artisan.md

```diff
@@ -87,3 +87,11 @@ php artisan p:info
 ```
 
 Displays a variety of panel information which can be used to check the configuation of this like database and email.
+
+### Update Panel
+
+```bash
+php artisan p:upgrade
+```
+
+Updates the panel to the latest version without having to manually copy the upgrade documentation.
```

Comment:
This should provide the available arguments similar to the other commands above since this page is for explaining the Artisan CLI. The upgrade command for only updating to latest is displayed in the upgrade documentation. You can see these by typing `php artisan help p:upgrade`

```bash
php artisan p:user:make {--user=www-data}
                        {--group=www-data}
                        {--url=https://example.com/panel.tar.gz}
                        {--release=latest}
                        {--skip-download}
```

Downloads a new archive for Pterodactyl and executes the normal upgrade commands